runtime/test-utils/CreateTestInstance.go

```diff
@@ -0,0 +1,22 @@
+package test_utils
```

Comment:
This package looks redundant

Reply:
I didnt use most of the stuff here. But it would be needed for other tests like metrics views API.